src/routes/+page.svelte

```diff
@@ -304,11 +307,20 @@
             textCopied = false;
         }, 3000);
     }
+
+    function handleBackNavigation(event) {
+        event.preventDefault();
```

Comment:
I know I wrote this, but I wonder if the `event.preventDefault()` call should be inside the if condition...